server/app-engine/gtfsrt.py

```diff
@@ -226,7 +245,10 @@ def get_alert_feed(datastore_client, agency):
     print('- agency: ' + agency)
 
     name = agency + '-alert-feed'
-    alert_feed = cache.get(name)
+    if not service_alert_ui:
```

Comment:
python has a trigraph construct, so you can simplify to: `alert_feed = cache.get(name) if use_cache else None`

Reply:
thats very nice, thanks